tests/common.go

```diff
@@ -329,6 +329,68 @@ func ValidateAndDestroy(contexts []*scheduler.Context, opts map[string]bool) {
 	})
 }
 
+// DescribeNamespace takes in the scheduler contexts and describes each object within the test context.
+func DescribeNamespace(contexts []*scheduler.Context) {
+	context(fmt.Sprintf("generating namespace info..."), func() {
+		Step(fmt.Sprintf("Describe Namespace objects for test %s \n", ginkgo.CurrentGinkgoTestDescription().TestText), func() {
+			for _, ctx := range contexts {
+				logrus.Info(Inst().S.Describe(ctx))
+			}
+		})
+	})
+}
+
+// CollectSupport creates a support bundle
+func CollectSupport() {
+	context(fmt.Sprintf("generating support bundle..."), func() {
+		Step(fmt.Sprintf("save all useful logs on each node"), func() {
+			nodes := node.GetWorkerNodes()
+			expect(nodes).NotTo(beEmpty())
+
+			for _, n := range nodes {
+				if !n.IsStorageDriverInstalled {
+					continue
+				}
+
+				// this task may hang so we wrapped in a timeout cmd
+				dumpTasksCmd := "timeout 10s bash -c \"echo t > /proc/sysrq-trigger\""
+				logrus.Infof("dump the list of current tasks and their information on %s", n.Name)
+				runCmd(dumpTasksCmd, n)
+
+				journalCmd := fmt.Sprintf("journalctl -l > ~/all_journal_%v.log", time.Now().Format(time.RFC3339))
+				logrus.Infof("saving journal output on %s", n.Name)
+				runCmd(journalCmd, n)
+
+				logrus.Infof("saving portworx journal output on %s", n.Name)
+				runCmd("journalctl -lu portworx* > portworx.log", n)
```

Comment:
Where is this executed on? You cannot write anything in / dir on OpenShift clusters, its gotta go to like /tmp or other available for write dir.

Reply:
@nikolaypopov as we agreed we're saving to /var/cores, which all systems have access to